### PR TITLE
Detect platform tizen and link profiler statically similar to Android.

### PR DESCRIPTION
**Pull request by joncham**

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -268,6 +269,11 @@ case "$host" in
                 	mono_cv_uscore=no
                 fi
 		case "$host" in
+		*-tizen-linux-*)
+			platform_tizen=yes
+			;;
+		esac
+		case "$host" in
```
Comment (lewurm) [inline configure.ac]: can't you remove those two lines?

Comment (joncham) [reply configure.ac]: @lewurm  I could. The original case if for architecture checks where this is a (sub)platform check. I wasn't sure if they should be merged or not. For example, if x86 is added to original check I likely want whatever is done there performed for Tizen as well.

Comment (lewurm) [reply configure.ac]: @joncham ah I see, that makes sense 👍

**Comment by alexrp** [comment]
Just to clarify: This does not actually mean that the profiler library is statically linked; it just makes it so that undefined symbols in the profiler library are resolved at link time rather than run time (which is fine).

**Pull request merged**